### BIG differences, this is diff of wrong files.
### CLUS "Function STRING"
### and CLHS topic selection http://clhs.lisp.se/Body/a_string.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:string.txt	2017-04-27 21:41:27.000000000 +0100
+++ saved-pages-clus/cl:functions:string.txt	2017-04-27 21:41:27.000000000 +0100
@@ -1,2 +1,1 @@
-STRING
-Please select which reference to STRING you intended
+Function STRING

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Syntax
+ string x → string
+Arguments and Values
+ x - a string, a symbol, or a character.
+ string - a string.
+Description
+Returns a string described by x; specifically
+ If x is a string, it is returned.
+ If x is a symbol, its name is returned.
+ If x is a character, then a string containing that one character is returned.
+ string might perform additional, implementation-defined conversions.
+Examples
+(string "already a string")
+→
+"already a string"
+(string 'elm)
+→
+"ELM"
+(string #\\c)
+→
+"c"
+Affected By
+None.
+Exceptional Situations
+In the case where a conversion is defined neither by this specification nor by the implementation, an error of type type-error is signaled.
+See Also
+ Function COERCE
+ Type STRING
+Notes
+coerce can be used to convert a sequence of characters to a string.
+prin1-to-string, princ-to-string, write-to-string, or format (with a first argument of nil) can be used to get a string representation of a number or any other object.
+\
\ No newline at end of file
